PowerEditor/src/Parameters.h

```diff
@@ -832,9 +832,9 @@ struct NppGUI final
 	TCHAR _defaultDirExp[MAX_PATH];	//expanded environment variables
 	generic_string _themeName;
 	MultiInstSetting _multiInstSetting = monoInst;
-	bool _fileSwitcherWithoutExtColumn = false;
+	bool _fileSwitcherWithoutExtColumn = true;

```

Comment:
I think this should be `false`.

Reply:
If my recall is good, the default value of withoutExt is true - to be verified in the old release.